week_1_basics_n_setup/README.md

```diff
@@ -100,6 +100,7 @@ If you have problems setting up the env, you can check this video:
 
 Did you take notes? You can share them here
 
+* [Notes from Candace Williams - 2023 Cohort](https://teacherc.github.io/data-engineering/2023/01/18/zoomcamp1.html)
```

Comment:
Can you please put your notes at the end?

Reply:
I just put my notes at the end, @alexeygrigorev.